src/poojap/BankingSystem.java

```diff
@@ -0,0 +1,71 @@
+package poojap;
+
+public class BankingSystem {
+
+	int creditCount;
+	int debitCount;
+	int currentBalance;
+	static int totalCreditCount;
+	static int totalDebitCount;
+	static int totalPrintBalCount;
+
+	void setCurrentBal(int currentBal){
+		currentBalance = currentBal;
+	}
+
+	void creditAmount(int creditAmount){
+		currentBalance = currentBalance + creditAmount;
+		creditCount++;
+		totalCreditCount++;
+		
+	}
+
+	void debitAmount(int debitAmt){
+		currentBalance = currentBalance - debitAmt;
+		debitCount++;
+		totalDebitCount++;
+		
+	}
+
+	void printBalance(){
+		System.out.println("Current Balance is : " +currentBalance);
```

Comment:
There is one requirement related this as well for individual user, that you missed here.
i want individual printTotalBalanace count as well.